Ruddarr/Views/Shared/InformationList.swift

```diff
@@ -76,8 +76,6 @@ struct InformationItem: Hashable {
 
         if let link = link {
             hasher.combine(link)
-        } else {
```

Comment:
I thought hashing in the zero was not needed (afaik we're not breaking anything by not doing it). If anything there could be a case where a non-nil value's hash resolves to `0` and we'll clash with that case (which is not an error, but still).

Feel free to reject this change, Im not certain Im correct here.